components/iam/telepresence.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# TODO(at) add config mounts
+
+telepresence --swap-deployment iam --expose 9002 --method vpn-tcp --run go run . run
```

Comment:
Do we really need this? What can you not do without it?

Reply:
I cannot e2e test a change of the flows in a preview-env.
Yes, we need that.

Reply:
Why do we need a preview environment in the first place?

Reply:
To experience the change we're working on.

Reply:
I'm deliberately trying to challenge some base assumptions here. What we have is an API which has some inherent behaviour, we should be able to write units tests which assert that it does X, given this input. We can then run a preview to validate it, but it doesn't feel like this is needed to iterate on the implementation.